lib/client/twofa/vip/api.go

```diff
@@ -1,4 +1,4 @@
-// Package vip does two factor authentication with Symantec VIP
+// Package vip does two factor authentication with Symantec VIP or Okta
```

Comment:
Either the Okta support should be in a new package, or this package should be renamed.